mobify/mobifyjs
# V2.0 modularization of deps

Status: **Opened for visibility**

Reviewers: **@jansepar**
JIRA: **https://mobify.atlassian.net/browse/PROD-1536**
Linked PRs: Not PRs, but new repos:
- https://github.com/mobify/mobifyjs-utils
- https://github.com/mobify/jazzcat-client
- https://github.com/mobify/imageresize-client
## Changes
- utils, imageresize and jazzcat have been made into bower components and are brought in that way
## How to test-drive this PR
- Checkout this branch, reset hard to ensure you don't have any old files around
- `bower install`
- Run `grunt build`, ensure build works properly and examine examples to ensure they are still working
